lib/openshift/node.rb

```diff
@@ -99,7 +99,12 @@ def is_master?(user: nil, cached: true, quiet: false)
     def is_windows_worker?(user: nil, cached: true, quiet: false)
       rr = raw_resource(user: user, cached: cached, quiet: quiet)
       rr.dig('metadata', 'labels', 'kubernetes.io/os') == "windows"
-    end    
+    end
+
+    def is_linux?(user: nil, cached: true, quiet: false)
```

Comment:
Perhaps unify the name to `is_linux_worker` ?

Reply:
Since master node also uses the label `kubernetes.io/os=linux` so it just can identify all linux nodes but cannot identify linux worker nodes. Maybe I can update the name as `is_linux_node`, WDYT?
It is different with the one `def is_windows_worker?` because windows node cannot be configured as master for now. Indeed I don't think `is_windows_worker` is accurate.

Reply:
Then feel free to update `is_windows_worker` to the proper one and make them unify.

Reply:
sure, updated all `is_windows_worker` to `is_windows`.